circuit_maintenance_parser/parser.py

```diff
@@ -346,6 +347,14 @@ def get_key_with_string(dictionary: dict, string: str):
                 return key
         return None
 
+    def get_llm_question(self):
+        """Return the LLM question."""
+        custom_llm_question = os.getenv("OPENAI_API_KEY")
```

Comment:
Doesn't look like the right variable to me :)

Is an environment variable the appropriate way to do this, given that the question can be a very long string with whitespace and newlines?

Reply:
well, maybe it's not the most user-friendly one, but the library has no config options so far... do you think it deserves another thought?

Reply:
I wonder if something like a path to a text file might be preferable. Not sure though.

Reply:
this could be an option. Actually, we could offer both (in precedence order):

1. `LLM_QUESTION_STR`
2. `LLM_QUESTION_FILEPATH`

Reply:
@glennmatthews I implemented this approach.